src/main/java/org/terasology/dialogs/action/NewDialogActionTypeHandler.java

```diff
@@ -17,33 +17,24 @@
 package org.terasology.dialogs.action;
 
 import java.util.Map;
+import java.util.Optional;
 
-import org.terasology.persistence.typeHandling.DeserializationContext;
-import org.terasology.persistence.typeHandling.PersistedData;
-import org.terasology.persistence.typeHandling.PersistedDataMap;
-import org.terasology.persistence.typeHandling.RegisterTypeHandler;
-import org.terasology.persistence.typeHandling.SerializationContext;
-import org.terasology.persistence.typeHandling.SimpleTypeHandler;
+import org.terasology.persistence.typeHandling.*;
```

Comment:
Please remove this star import. You can also change your IntelliJ glob import setting to a very high number to prevent IntelliJ from doing this automatically :)